pkgs/development/python-modules/mtcnn/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, opencv3
+, tensorflow
+, keras
+}:
+
+
+buildPythonPackage rec {
+  pname = "mtcnn";
+  version = "0.1.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256:0arhh6ir8vrmscgg0i9yi6qz6g7fz10r022a7nw2rrjbb1s755fh";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "opencv-python>=4.1.0" ""
+  '';
+
+  propagatedBuildInputs = [
+    opencv3
+    keras
+    tensorflow
+  ];
+
```

Comment:
Could you please enable tests with `pytestCheckHook` and `pythonImportsCheck`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

My concern is that this package has not been maintained in over two years.
As you know, machine learning packages change too fast, and if we can't confirm that it works, it will be scrapped as soon as it is merged.